test/integration/cljdoc_analyzer/test_helper.clj

```diff
@@ -10,6 +10,19 @@
                   (str project "/" project))]
     (str prefix "/" project "/" version "/cljdoc.edn")))
 
+(defn- filter-namespace
+  "Filter undesired namespace in a analysis map.
+  Useful when arguments of function are generated by gensyms: no reproducible analysis"
+  [analysis-map language namespace-to-filter]
+  (update analysis-map :analysis
+          (fn [map-clj-cljs]
+            (update map-clj-cljs language
+                    (fn [list-namespace]
+                      (filter
+                       (fn [item]
+                         (prn item)
+                         (not= (str (:name item)) namespace-to-filter))
+                       list-namespace))))))
```

Comment:
This could be simplified by using `update-in` and potentially supplying the namespaces to remove as a set:
```clj
(defn- filter-namespaces
  "Filter undesired namespaces in a analysis map.
  Useful when arguments of function are generated by gensyms: no reproducible analysis"
  [analysis-map language remove-ns?]
  (update-in analysis-map
             [:analysis language]
             #(remove (fn [ns] (remove-ns? (:name ns))) %)))

(filter-namespaces {:analysis {"cljs" [{:name "a"} {:name "b"}]}}
                   "cljs"
                   #{"a"})
;; => removes "a"
```

Reply:
You're completely right. That's why I would like to code on a project with review from more experienced people: get the good automatism :+1: 

I upgraded the code in my last commit.